melosys-eessi-app/src/main/resources/application-local-mock.properties

```diff
@@ -40,7 +40,7 @@ KAFKA_TRUSTSTORE_PATH=/dev/zero
 KAFKA_CREDSTORE_PASSWORD=pwd
 
 KAFKA_AIVEN_SEDMOTTATT_TOPIC=eessibasis-sedmottatt-v1-local
-KAFKA_AIVEN_SEDSENDT_TOPIC=eessibasis-sedsendt-v1
+KAFKA_AIVEN_SEDSENDT_TOPIC=eessibasis-sedsendt-v1-local
```

Comment:
Skal denne endringen være med?

Reply:
Synest det ga lite mening å kalle den local for mottatt men ikke for sendt. Men kan kansje være lurt å endre den tilbake slik att folk ikke får problemer med det lokale utviklingsmiljøet sitt